sdk/batch/Azure.Compute.Batch/tests/Integration/EndToEndIntegrationTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Compute.Batch.Tests.Infrastructure;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Compute.Batch.Tests.Integration
+{
+    internal class EndToEndIntegrationTests : BatchLiveTestBase
```

Comment:
Can we get rid of this unused class?

Reply:
removed, I didn't clean these up before, good catch